src/test/ui/consts/offset_of_ub.rs

```diff
@@ -0,0 +1,44 @@
+#![feature(const_raw_ptr_deref)]
+#![feature(const_ptr_offset_from)]
+#![feature(ptr_offset_from)]
+
+#[repr(C)]
+struct Struct {
+    data: u8,
+    field: u8,
+}
+
+pub const DIFFERENT_ALLOC: usize = {
+    //~^ NOTE
```

Comment:
Maybe add some keyword from the error message, to be sure we are testing the right thing here.
(People to `--bless` relentlessly, so I am not convinced that someone would notice there when the error message gets a bit different.)

Reply:
I can't... the error message's span is in the libstd... I could maybe try to add `error-pattern` comments

Reply:
Dang. Well I guess that has to be good enough then.